modules/nixos/shared.nix

```diff
@@ -29,10 +29,10 @@
   nix.gc =
     {
       automatic = true;
-      options = lib.mkForce ''--max-freed "$((32* 1024**3 - 1024 * $(df -P -k /nix/store | tail -n 1 | ${pkgs.gawk}/bin/awk '{ print $4 }')))"'';
+      options = lib.mkForce ''--max-freed "$((48* 1024**3 - 1024 * $(df -P -k /nix/store | tail -n 1 | ${pkgs.gawk}/bin/awk '{ print $4 }')))"'';
     }
-    // lib.optionalAttrs pkgs.stdenv.isLinux (lib.mkForce {dates = "*:30";})
-    // lib.optionalAttrs pkgs.stdenv.isDarwin {interval.Minute = 30;};
+    // lib.optionalAttrs pkgs.stdenv.isLinux (lib.mkForce {dates = "*:17";})
+    // lib.optionalAttrs pkgs.stdenv.isDarwin {interval.Minute = 17;};
```

Comment:
i tried to look at the logs and see when GC triggers but couldn't find any occurrence of it under `/var/log`. any idea where this is logged on macos?

Reply:
`log show | grep nix-gc` shows that it is running every 17 minutes

Reply:
Oops, I meant it is running every hour on minute 17, like expected

Reply:
(we're commenting on the macos line)

trying this on macos-02:

```
administrator@Administrators-Mac-mini> log show | grep nix-gc                                                               ~
log: too many arguments
```